rcl/src/rcl/event.c

```diff
@@ -179,7 +181,7 @@ rcl_event_fini(rcl_event_t * event)
 rmw_event_t *
 rcl_event_get_rmw_handle(const rcl_event_t * event)
 {
-  if (NULL == event) {
+  if (!rcl_event_is_valid(event)) {
     return NULL;  // error already set
```

Comment:
I like that the comment `// error already set` was wrong, but now it's correct :joy: